pymc_marketing/mmm/transformers.py

```diff
@@ -715,3 +715,37 @@ def tanh_saturation_baselined(
     Developed by Max Kochurov and Aziz Al-Maeeni doing innovative work in `PyMC Labs <pymc-labs.com>`_.
     """
     return gain * x0 * pt.tanh(x * pt.arctanh(r) / x0) / r
+
+
+def michaelis_menten(
+    x: Union[float, np.ndarray, npt.NDArray[np.float64]],
+    alpha: Union[float, np.ndarray, npt.NDArray[np.float64]],
+    lam: Union[float, np.ndarray, npt.NDArray[np.float64]],
+) -> Union[float, Any]:
+    """
```

Comment:
It would be great to use latex for the equation and ideally a plot as in https://pymc-marketing--541.org.readthedocs.build/en/541/api/generated/pymc_marketing.mmm.transformers.logistic_saturation.html 🙏